README.md

```diff
@@ -1367,6 +1366,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Pritam Sarbajna](https://github.com/PritamSarbajna)
   - [Prithviraj Biswas](https://github.com/prithvirajbytes)
   - [Priyansh Mehta](https://github.com/Priyansh61)
+  - [Priyankar Pal](https://github.com/priyankarpal)
```

Comment:
There's a GitHub conflict now. You shouldn't remove a line. As you can see with *Pablo Lorenzo* is in line 1299 now, it should be in line 1300. To add your name just a new space. And it should be before *Priyansh Mehta*. Please see the origin `README.md` for the reference.

Reply:
Check I think Now it's ok.